docs/output.md

```diff
@@ -392,6 +392,21 @@ By default, only the raw bins (and unbinned contigs) from the actual binning met
 
 ⚠️ Due to ability to perform downstream QC of both raw and refined bins in parallel (via `--postbinning_input)`, bin names in DAS Tools's `*_allBins.eval` file will include `Refined`. However for this particular file, they _actually_ refer to the 'raw' input bins. The pipeline renames the input files prior to running DASTool to ensure they can be disambiguated from the original bin files in the downstream QC steps.
 
+### Tiara
+
+Tiara is a contig classifier that identifies the domain (prokarya, eukarya) of contigs within a file. This is used in this pipeline to find the most likely domain classification of each bin or unbin based on its contig identities.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `GenomeBinning/Tiara/`
+  - `[assembler]-[sample/group].tiara.txt` - Tiara output classifications (with probabilities) for all contigs within the specified sample/group assembly
+  - `[assembler]-[binner]-[bin_type]-[sample/group].binclassification.tsv` - Final classification of each bin/unbin based on the classification probabilities of each contig within the bin.
+  - `log/log_[assembler]-[sample/group].txt` - log file detailing the parameters used by the Tiara model for contig classification.
+- `GenomeBinning/tiara_summary.tsv` - Summary of Tiara domain classification for all bins.
+
+</details>
+
```

Comment:
I always like to have a sentence or two after the list of files of what you could use the subsequent files for

e.g. 'Typically you would use `tiara_summary.tsv` as the primary file to allow you to pull out XYZ, whereas only (`binclassification.tsv`) if you need XYZ additional information'